week-1/L-functions-nested/exercise2.js

```diff
@@ -3,3 +3,16 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function upperCase(name) {
+  return name.toUpperCase();
+}
+console.log(upperCase(mentor1));
+
+console.log(upperCase(mentor2));
+
+console.log(upperCase(mentor3));
+
```

Comment:
again, good job , but the exercise in fact asks for 2 functions: one to produce a greeting (not just the name).

Reply:
"Exercise 2
In exercise2.js you have been provided with the names of some mentors. Write a program that logs a shouty greeting to each one.
Your program **should include a function** that spells their name in uppercase, **and a function that creates a shouty greeting**."